apps/web-giddh/src/app/settings/integration/setting.integration.component.ts

```diff
@@ -307,13 +307,6 @@ export class SettingIntegrationComponent implements OnInit, AfterViewInit {
             }
         });
 
-        this.store.pipe(select(s => s.session.currentCompanyCurrency), takeUntil(this.destroyed$)).subscribe(res => {
-            if (res) {
-                //This is not using no due to gocardless
```

Comment:
revert this code only remove for only gocardless

Reply:
Done